### PR TITLE
api services & non-relative module imports

### PR DESCRIPTION
**Pull request by artyomHiqo**

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -123,6 +130,7 @@
           }
         }
       }
-    }},
+    }
+  },
   "defaultProject": "angular-trello-test"
-}
+}
```
Comment (ElizavetaSelivanova) [inline angular.json]: Add empty line

File: src/app/app-routing.module.ts [hunk 2]
```diff
@@ -1,7 +1,27 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 
-const routes: Routes = [];
+import { PageNotFoundComponent } from './page-not-found/page-not-found.component';
+
+const routes: Routes = [
+  {
+    path: '',
+    redirectTo: '/dashboards',
```
Comment (ElizavetaSelivanova) [inline src/app/app-routing.module.ts:10]: Did you add a guard?
(`canActivate`)

File: src/app/components/header/header.component.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<mat-toolbar color="primary" class="navbar">
+
+  <a>Trello</a>
```
Comment (ElizavetaSelivanova) [inline src/app/components/header/header.component.html]: Incorrect element =)

File: src/app/components/header/header.component.html [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+  <div class="navbar-icons">
+    <div>
```
Comment (ElizavetaSelivanova) [inline src/app/components/header/header.component.html]: remove empty div, or add class)

File: tsconfig.json [hunk 5]
```diff
@@ -2,7 +2,7 @@
 {
   "compileOnSave": false,
   "compilerOptions": {
-    "baseUrl": "./",
+    // "baseUrl": "./",
```
Comment (ElizavetaSelivanova) [inline tsconfig.json]: remove unused comment

File: tsconfig.json [hunk 6]
```diff
@@ -15,6 +15,11 @@
     "lib": [
       "es2018",
       "dom"
-    ]
+    ],
+    "baseUrl": "src",
+    "paths": {
+      "@app-services/*": ["app/core/services/*"],
+      "@app-environments/*": ["app/environments/*"],
```
Comment (ElizavetaSelivanova) [inline tsconfig.json:21]: you can add just @app =)

Comment (artyomHiqo) [reply tsconfig.json:21]: i know, but such naming is more understandable for me)

**Pull request merged**